packages/connectivity/src/scp-cf/destination/http-proxy-util.ts

```diff
@@ -19,33 +19,35 @@ const logger = createLogger({
 });
 
 /**
- * Determines the proxy strategy. If noProxy is set the ProxyConfiguration in the destination is omitted.
- * For onPremProxy or internetProxy the connectivity service or environment variables are checked to fill the {@link ProxyConfiguration}.
- * @param destination - from which the proxy strategy is derived.
- * @returns ProxyStrategy possible values are noProxy, internetProxy or onPremProxy.
  * @internal
+ * Determines the proxy strategy. If the 'no_proxy' env variable is set, the `ProxyConfiguration` in the destination is omitted.
+ * For the 'on-premise' and 'internet' proxy strategies the connectivity service or environment variables are checked to fill the `ProxyConfiguration`.
+ * @param destination - Destination to derive the proxy strategy from.
+ * @returns The proxy strategy for the given destination.
  */
-export function proxyStrategy(destination: Destination): ProxyStrategy {
+export function proxyStrategy(
+  destination: Destination
+): 'no-proxy' | 'on-premise' | 'internet' | 'private-link' {
```

Comment:
[q] Should we not create these as types?

Reply:
I added a `ProxyStrategy` type.